assets/js/payment-methods-demo/payment-methods/index.js

```diff
@@ -1,41 +1,25 @@
 /**
- * Internal dependencies
+ * External dependencies
  */
-import { paypalSvg } from './paypal';
-import { ccSvg } from './cc';
-
-const PaypalActivePaymentMethod = () => {
-	return (
-		<div>
-			<p>This is where paypal payment method stuff would be.</p>
-		</div>
-	);
-};
+import { __ } from '@wordpress/i18n';
 
-const CreditCardActivePaymentMethod = () => {
-	return (
-		<div>
-			<p>This is where cc payment method stuff would be.</p>
-		</div>
-	);
-};
+/**
+ * Internal dependencies
+ */
+import { PAYMENT_METHOD_NAME, StripeCreditCard } from './stripe';
+import { stripePromise } from '../stripe-utils';
 
-export const paypalPaymentMethod = {
-	id: 'paypal',
-	label: <img src={ paypalSvg } alt="" />,
-	stepContent: <div>Billing steps</div>,
-	activeContent: <PaypalActivePaymentMethod />,
-	edit: <PaypalActivePaymentMethod />,
-	canMakePayment: Promise.resolve( true ),
-	ariaLabel: 'paypal payment method',
-};
+const EditPlaceHolder = () => <div>TODO: Card edit preview soon...</div>;
 
-export const ccPaymentMethod = {
-	id: 'cc',
-	label: <img src={ ccSvg } alt="" />,
-	stepContent: null,
-	activeContent: <CreditCardActivePaymentMethod />,
-	edit: <CreditCardActivePaymentMethod />,
-	canMakePayment: Promise.resolve( true ),
-	ariaLabel: 'credit-card-payment-method',
+export const stripeCcPaymentMethod = {
+	id: PAYMENT_METHOD_NAME,
+	label: (
+		<strong>
+			{ __( 'Credit/Debit Card', 'woo-gutenberg-products-block' ) }
+		</strong>
+	),
+	activeContent: <StripeCreditCard />,
+	edit: <EditPlaceHolder />,
+	canMakePayment: stripePromise,
```

Comment:
The naming of this one is slightly odd. I assume this is basically whether or not the method will be enabled or visible? Remember some gateways (offline, invoice etc) don't technically take payment.

Reply:
It's for a consistent api, because some payment methods can only make a payment given certain conditions on the environment (browser in use, configured properly, third party library loaded etc).  This allows for a consistent way for gateways to indicate they can make payment.  For offline, or invoice payment methods they could just do `Promise.resolve( true )` as the value for this field.